src/main/java/com/google/api/generator/engine/ast/LineComment.java

```diff
@@ -32,8 +33,15 @@ public static LineComment withComment(String comment) {
   @AutoValue.Builder
   public abstract static class Builder {
     public abstract Builder setComment(String comment);
+    // Private accessor.
+    abstract String comment();
 
-    public abstract LineComment build();
+    public abstract LineComment autoBuild();
+
+    public LineComment build() {
+      setComment(CommentEscaper.escape(comment()));
```

Comment:
Does this need escaping?

Reply:
yes, there is also case that LineComment contains special characters \t\b\r etc, if they are not escaped, when we call java formatter, it will throw error of FormatException.